jenkins/model_output_processing_Jenkinsfile

```diff
@@ -37,6 +37,7 @@ pipeline {
           PATTERN="*.nc"
           aws s3 sync s3://owi-common-resources/resources/application/nhm-dev/output/data/ . --exclude "*" --include "${PATTERN}"
           aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/model_output/${TIER}/ . --exclude "*" --include "all_quantiles.rds"
+          aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/model_output/${TIER}/variables/ . --exclude "*" --include "*.rds" --include "*.csv"
```

Comment:
gets the new quantiles for each variable and the csv with maxes to use later in the tests